packages/cashier-v2/tailwind.config.ts

```diff
@@ -0,0 +1,7 @@
+import type { Config } from 'tailwindcss';
+import QuillTailwindConfig from '@deriv/quill-design/quill-tailwind/tailwind.config.cjs';
```

Comment:
tailwind :D :D :D on the one hand, I really like using tailwind in personal small project, on the other hand, I question how its gonna scale up in large and complex application :D it seems to me to be one of those concpets which are really appealing at the beginning or for small scale projects, but do not necesarily scale up well,  but, I guess only time will tell how it plays on large scale :)

Reply:
let see @wojciech-deriv
hope that all should be fine 😉